pkg/engine/internal/imageverifier.go

```diff
@@ -324,7 +341,14 @@ func (iv *ImageVerifier) verifyImage(
 		return engineapi.RuleError(iv.rule.Name, engineapi.ImageVerify, fmt.Sprintf("failed to add image to context %s", image), err), ""
 	}
 	if len(imageVerify.Attestors) > 0 {
-		if !matchImageReferences(imageVerify.ImageReferences, image) {
+		if !matchReferences(imageVerify.ImageReferences, image) {
+			return nil, ""
+		}
+
+		if matchReferences(imageVerify.SkipImageReferences, image) {
+			iv.logger.Info("skipping image reference", "image", image, "policy", iv.policyContext.Policy().GetName(), "ruleName", iv.rule.Name)
+			iv.ivm.Add(image, engineapi.ImageVerificationSkip)
+			engineapi.RuleWarn(iv.rule.Name, engineapi.ImageVerify, fmt.Sprint("skipping image reference", "image", image, "policy", iv.policyContext.Policy().GetName(), "ruleName", iv.rule.Name))
```

Comment:
@JimBugwadia @realshuting @eddycharly, should we return a warn/skip/no response when an excluded image is skipped?